src/main/java/org/opensearch/ad/util/ParseUtils.java

```diff
@@ -714,4 +715,13 @@ public static boolean listEqualsWithoutConsideringOrder(List<String> list1, List
         }
         return Objects.equals(set1, set2);
     }
+
+    public static double[] parseDoubleArray(XContentParser parser) throws IOException {
```

Comment:
Why is this method needed?

Reply:
It is used in parse double array out of XContent in AnomalyResult.parse method.